pyoozie/__init__.py

```diff
@@ -1,4 +1,21 @@
 # Copyright (c) 2017 "Shopify inc." All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
 
+from pyoozie.coordinator import Coordinator, ExecutionOrder
+from pyoozie.tags import Parameters, Configuration, Credentials, Shell, SubWorkflow, GlobalConfiguration, \
+    Email, IdentifierTooLongError
+from pyoozie.builder import WorkflowBuilder, CoordinatorBuilder
+
 __version__ = '0.0.0'
```

Comment:
When should we update this; on every intended release? Every PR would become a bit tedious.